.github/workflows/PublishWidgetData.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish widget data
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+jobs:
+  publish-data:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We could maybe add a step here where we check if anything has changed in certain paths in the last 24 hours, so we limit the Github resources being used.


Alternatively, using a PR or commit as the trigger of this action enables us to filter for changes in certain paths. But that probably also mean the action runs multiple times a day and creates multiple PR's

Reply:
Yea, I'm still in doubt. Nightly has the nice benefit that it doesn't run very often. We could use Push to Master as trigger, for specific paths only, I think? Does that trigger when you merge a PR?
The resulting json file won't change very often, and when it doesn't change, it doesn't create a PR. So I don't think we have to worry about creating multiple PRs.

Reply:
I was a bit worried about the number of PRs as well, but if there's no PR if there's no change in the resulting json, then that's perfect 😄

Reply:
True, the json file will probably not change that much. Yes I'm pretty sure Push to Master is triggered when a PR is merged :)

Reply:
I've changed it to `push on master`